pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -12,23 +11,24 @@ stdenv.mkDerivation rec {
     sha256 = "1wjmfvbsq3126z51f2ivj85cjmkrzdm2acqsiyqs57qga2g6w5p9";
   };
 
-  buildInputs = [ stdenv makeWrapper ];
+  buildInputs = [ makeWrapper ];
 
   installPhase = ''
     mkdir -p $out/bin
     mkdir -p $out/share/man/man1
     cp ${src}/rmount.man $out/share/man/man1/rmount.1
-    cp ${src}/rmount.bash $out/bin/rmount-noenv
+    cp ${src}/rmount.bash $out/bin/rmount
     cp ${src}/config.json $out/share/config.json
-    chmod +x $out/bin/rmount-noenv
+    chmod +x $out/bin/rmount
 
-    makeWrapper $out/bin/rmount-noenv $out/bin/rmount \
-    --prefix PATH : ${lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
+    wrapProgram $out/bin/rmount --prefix PATH : ${lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
```

Comment:
`lib` can be removed from the parameters up top, and `stdenv.lib.makeBinPath` used here.

In Nixpkgs there are 205 `stdenv.lib.makeBinPath` and 109 `lib.makeBinPath`, making the former preferred.

If you're curious, `stdenv.lib` is the same as `lib`.

```
nix-repl> stdenv.lib == lib
[...]
true
```